src/styles/main.scss

```diff
@@ -1,26 +1,26 @@
-/* ------ VI colors, done by Philip ------------- */
+/* ------ VI 2.0 colourrs */
```

Comment:
*colours*

Reply:
Damn me xD something else I should change?

Reply:
other than that seems fine